packages/dialtone-css/lib/build/less/components/scrollbar.less

```diff
@@ -0,0 +1,25 @@
+.custom-scrollbars {
+  .os-theme-dark {
```

Comment:
I'm confused about the `.os-theme-dark` part. I noticed the rendered markup always has  `os-theme-dark` in there regardless of system setting or Dialtone theme toggle.

Reply:
The theme dark is part of what the plugin adds, because it allows changing themes. Since we are not using that feature (we use our own tokens that adapt to the theme), is always theme dark for the plugin. I needed to add that selector because of specificity reasons.